scripts/gen_reference_page.py

```diff
@@ -0,0 +1,30 @@
+"""
+Automatically add all OpenSquirrel Python modules to docs/reference.md whenever the mkdocs documentation is built.
+"""
+
+from pathlib import Path
+
+import mkdocs_gen_files
+
+repo = Path(__file__).parent.parent
+opensquirrel = repo / "opensquirrel"
+reference_file = repo / "docs" / "reference.md"
+
+reference_md_content = "<!--- This file was automatically generated by scripts/gen_reference_page.py --->\n\n"
+for path in sorted(opensquirrel.rglob("*.py")):
+    module_path = path.relative_to(repo).with_suffix("")
+
+    print(module_path)
```

Comment:
Do we want to introduce `logging` for this, in stead of print statements? If so, we can add a TODO in the places and create an issue to pick this up later.

Reply:
This `print` was left accidentally, I didn't intend to leave it
now removed
Nevertheless, logging can be nice, so we can create the issue.